app-datepicker.html

```diff
@@ -1087,12 +1095,17 @@
           if (_maxDateObj) {
             _isMoreThanMaxDate = _currentDate.getTime() > _maxDateObj.getTime();
           }
+          _isDisableDates = this.disableDates.some(function(date) {
+            var _dateObj = this._convertDateStringToDateObject(date);
+            return _dateObj && _currentDate.getTime() == _dateObj.getTime();
```

Comment:
Good catch, completely forgot I changed that to a "==" while I was testing.
Updated to PR with the change.

Thanks for taking the time to look at this!